CHANGELOG.md

```diff
@@ -9,20 +9,21 @@ _Dec 8, 2022_
 
 We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
 
-- 🚀 Add dragging support for `DateRangePicker` (#6763) @LukasTy
-- ✨ Add missing lazy-loading feature to `DataGridPremium` (#7124) @m4theushw
+- 🚀 Add dragging support for the new Date Range Picker (`NextDateRangePicker`) (#6763) @LukasTy
+- ⚡️ Improve performance of `DayCalendar` (#7066) @flaviendelangle
```

Comment:
```suggestion
- ⚡️ Improve performance of the `day` view (#7066) @flaviendelangle
```

This component is private, let's use the view which is how devs can enable it :+1: